### PR TITLE
uiux improvements- bianca

### PR DESCRIPTION
**Pull request by biancasap**

### CODE REVIEW

File: src/main/webapp/dashboard.js [hunk 1]
```diff
@@ -259,30 +259,45 @@ function getEnvs() {
 // Display the queue redirect link and environments once page loads
 function onload() {
   setRedirect();
+
   firebase.auth().onAuthStateChanged(function(user) {
+    user.getIdToken().then((token) => {
```
Comment (nmittu) [inline src/main/webapp/dashboard.js:264]: You can use the `getToken()` helper function in `auth.js`

File: src/main/webapp/dashboard.js [hunk 2]
```diff
@@ -259,30 +259,45 @@ function getEnvs() {
...
-        } 
-        // Redirect to home page if not logged in
-        else {
-          console.log("User is not logged in");
-          window.location.href = "/";
-        }
-      });
-}
\ No newline at end of file
+    firebase.auth().onAuthStateChanged(function(user) {
```
Comment (nmittu) [inline src/main/webapp/dashboard.js:285]: Same here

File: src/test/java/com/google/sps/servlets/GetUserRoleTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetUserRoleTest {
```
Comment (bncollender) [inline src/test/java/com/google/sps/servlets/GetUserRoleTest.java:32]: There should probably also be a test for getting the student role

Comment (biancasap) [reply src/test/java/com/google/sps/servlets/GetUserRoleTest.java:32]: just added!

**Pull request merged**